crates/ruff_wasm/README.md

```diff
@@ -0,0 +1,41 @@
+# Ruff API
+
+[**Docs**](https://docs.astral.sh/ruff/) | [**Playground**](https://play.ruff.rs/)
```

Comment:
We should add a very big warning saying that this API is experimental and that it can change at any time.